apps/mdreader/app/components/panels/KnowledgePanel.tsx

```diff
@@ -112,11 +114,27 @@ const KnowledgePanel: React.FC<KnowledgePanelProps> = ({
   const onSave = async (data: KnowledgeForm & { metadata?: URLMetadata }) => {
     const metadata = data.metadata;
     const image = metadata?.image || metadata?.['og:image'];
+    let blurhash;
+
+    if (image) {
+      const { blurhash: encodedBlurHash, error } = await encodeImageToBlurhash(
+        image
+      );
+
+      if (encodedBlurHash) {
+        blurhash = encodedBlurHash;
+      }
+
+      if (error) {
+        console.error('Blurhash error: ', error);
+      }
```

Comment:
**Suggestion:** Consider handling the error from 'encodeImageToBlurhash' function properly. Instead of just logging it, you could show a user-friendly error message.
```suggestion
      if (error) {
        toast({
          title: 'Error',
          description: 'Failed to encode image to blurhash.',
          status: 'error'
        });
      }
```